examples/views/dropdown.py

```diff
@@ -0,0 +1,38 @@
+import discord
+from discord.ext import commands
+
+class Bot(commands.Bot):
+    def __init__(self):
+        super().__init__(command_prefix=commands.when_mentioned_or('$'))
+
+    async def on_ready(self):
+        print(f'Logged in as {self.user} (ID: {self.user.id})')
+        print('------')
+
+#Gives us a view containing a dropdown menu
+class Dropdown(discord.ui.View):
+    def __init__(self):
+        super().__init__()
+
+    #Send a message pinging the user with their selected choice
+    @discord.ui.select(options=[
+        discord.SelectOption(label="Red"), 
```

Comment:
might be good to have examples for description/emoji/placeholder?

Reply:
Would that work better in a different example or should I try to implement it here?

Reply:
you can have multiple commands/things in one example, i would have another command, that has emoji and placeholder

Reply:
Sounds good, I'll work on it.